cas-server-support-x509/pom.xml

```diff
@@ -54,10 +54,10 @@
       <scope>test</scope>
     </dependency>
     <dependency>
-      <artifactId>vt-crypt</artifactId>
-      <groupId>edu.vt.middleware</groupId>
-      <version>2.1.4</version>
-    </dependency>
+	  <groupId>org.cryptacular</groupId>
```

Comment:
One other comment: I think this should be defined in the root pom, and then referenced down here. Since we have other modules that use the same dependency, it's good to make them all point to the same thing and make that explicit.